### DISCUSSION TITLE
Version 2 pre-release

### DISCUSSION THREAD

**Discussion by jonghall** [jonghall/ibmcloud_vpc_ansible_dynamic_inventory#13] [Announcements]
Version 2 Changes
- Inventory queried using IBM Cloud Virtual Private Cloud (VPC) Python SDK Version 0.10.0
- Addition of IBM Cloud VPC Baremetal resource type
- Addition of group by resource type
- Addition of Placement Groups and the ability to group by placement group
- Addition of Dedicated Host field if present
- Addition of GPU fields
- Ability to query all regions by default or specify specific regions
- Removal of [api] section which is not needed by the SDK
- Movement of optional region variable to the [ibmcloud] section
- Varoious other fixes

**Comment by ghost** [comment]
Just from a rough read: @jonghall did moving the settings to an .ini file instead of environment variables essentially break AWX/Tower ability to control the script?
